ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py

```diff
@@ -2527,3 +2530,36 @@ def test_tensorflow_in_top_k(
         pred=x[1],
         k=k,
     )
+
+# l2_normalize
+@st.composite
+def _shape_and_axis(draw):
+    shape = draw(helpers.get_shape(allow_none=False, min_num_dims=1, max_num_dims=4))
+    axis = draw(helpers.axes(shape))
+    return shape, axis
+@handle_frontend_test(
+    fn_tree="tensorflow.math.l2_normalize",
+    shape_and_axis=_shape_and_axis(),
+    #eps=st.floats(min_value=0, max_value=1),
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=helpers.get_dtypes("float")),
+    test_with_out=st.just(False),
+)
+def test_tensorflow_l2_normalize(
+    *,
+    dtype_and_x,
+    shape_and_axis,
+    #eps,
+    frontend,
+    test_flags,
+    fn_tree,
+    on_device,
+):
+    import pdb
```

Comment:
I think you were using a debugger here. This code should be removed after local testing is finished.